lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -40,11 +40,21 @@ public function index($parameters) {
   public function show($id) {
     try {
       $reportbackItem = ReportbackItem::get($id);
-      $reportbackItem = $this->removeUnauthorizedResults($reportbackItem);
+
+      if (!user_access('administer modules')) {
+        $reportbackItem = $this->removeUnauthorizedResults($reportbackItem, TRUE);
+      }
+
       $reportbackItem = services_resource_build_index_list($reportbackItem, 'reportback-items', 'id');
     }
     catch (Exception $error) {
-      http_response_code('404');
+      if ($error->getMessage() === 'Access denied.') {
+        http_response_code('403');
+      }
+      else {
+        http_response_code('404');
+      }
```

Comment:
ooo that's really cool to know you can just add it as a second arg! The only thing about moving it back to the class is that sometimes, we don't want the code to be thrown ([explained in this issue](https://github.com/DoSomething/phoenix/pull/6280)) so we moved it to the Transformer level.